Userland/Libraries/LibGfx/ImageFormats/ILBMLoader.cpp

```diff
@@ -125,27 +112,23 @@ static ErrorOr<void> decode_iff_ilbm_header(ILBMLoadingContext& context)
     return {};
 }
 
-static ErrorOr<Vector<u32>> decode_cmap_chunk(Chunk cmap_chunk)
+static ErrorOr<FixedArray<Color>> decode_cmap_chunk(Chunk cmap_chunk)
 {
     auto data = cmap_chunk.data;
-    const u8 size = (u8)(data.size() / 3);
-    Vector<u32> color_table;
+    u8 const size = (data.size() / 3);
+    FixedArray<Color> color_table = TRY(FixedArray<Color>::create(size));
 
     for (int i = 0; i < size; ++i) {
-        u32 color = (data[i * 3] << 16) | (data[(i * 3) + 1] << 8) | data[(i * 3) + 2];
-        color |= 0xff000000;
-        color_table.append(color);
+        color_table[i] = Color(data[i * 3], data[(i * 3) + 1], data[(i * 3) + 2], 255);
```

Comment:
`Color` has a construct that only takes three parameters

Reply:
I saw that, but I was already thinking about using it when I'll have implemented mask/transparent color. But I guess I can switch to this constructor when I'll be implementing that.

Reply:
You will need to modify the line anyway, no need to bloat it right now